pyproject.toml

```diff
@@ -14,11 +14,12 @@ asyncpg = "^0.29.0"
 regtech-api-commons = {git = "https://github.com/cfpb/regtech-api-commons.git"}
 regtech-data-validator = {git = "https://github.com/cfpb/regtech-data-validator.git"}
 regtech-regex = {git = "https://github.com/cfpb/regtech-regex.git"}
+boto3 = "~1.34.0"
```

Comment:
why the downgrade? fsspec related?

Reply:
Yeah, with the boto3 and fsspec tie in in the data-validator.  But if I remove that to have the download send back a df, this dependency goes away